kolibri/core/auth/apps.py

```diff
@@ -13,3 +13,10 @@ class KolibriAuthConfig(AppConfig):
     def ready(self):
         from .signals import cascade_delete_membership  # noqa: F401
         from .signals import cascade_delete_user  # noqa: F401
+
+        from kolibri.core.auth.sync_event_hook_utils import (
+            register_sync_event_handlers,
+        )  # noqa: F401
+        from morango.api.viewsets import session_controller  # noqa: F401
+
+        register_sync_event_handlers(session_controller)
```

Comment:
I wasn't sure whether this was the best place to put this, but... I couldn't think of a better place. Suggestions welcome, or maybe this is ok.